.github/workflows/scala.yml

```diff
@@ -21,5 +21,5 @@ jobs:
         distribution: "temurin"
         java-version: ${{ matrix.java }}
     - name: Run tests
-      run: sbt -v mimaReportBinaryIssues test unidoc
+      run: sbt -v test unidoc
```

Comment:
Isn't it possible to exclude errors explicitly on sbt side instead of disabling MiMa check in CI? We can remove the exclude configuration after 3.1.0 release.

Reply:
Thanks for reminded me that, I had forgot that MiMa had to be restored.
This is now done in the last commit.